scripts/scan.sh

```
@@ -1 +1,11 @@
 # TODO: write the script
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)